src/shogun/io/SGIO.cpp

```diff
@@ -189,6 +196,7 @@ void SGIO::progress(
 			if (current_val >= max_val-1)
 			{
 				v = 100;
+				last_progress=v-1e-6;
```

Comment:
Core developer's will is law ;). Jokes aside, I will change it to 0. As I said, it the same :+1: